src/components/sources/SourceWMTS.vue

```diff
@@ -135,7 +135,7 @@ export default {
     },
     tileMatrixPrefix: {
       type: String,
-      default: ''
+      default: "",
```

Comment:
Note that this change isn't relevant to this PR, but `prettier` changed it automatically when I was cleaning up the code.